.github/workflows/integration.yml

```diff
@@ -35,10 +35,6 @@ jobs:
           neovim: true
           version: nightly
 
-      - name: Install plenary
```

Comment:
now we load plenary from lua: https://github.com/kylechui/nvim-surround/pull/246/files#diff-dff985f3ac32aac36a5ec14df2bba7e6a8c063293e4ea1a83cb1587d0a3eebbaR11